tasks/makeSearchIndex.js

```diff
@@ -5,8 +5,7 @@ const {parseProjects} = require('./utils/parseProjects')
 if (require.main !== module) {
   module.exports = function(config) {
     const targets = ['build/.temp/search_index.json']
-
-    const boards = parseProjects(config, true)
+    const boards = parseProjects(config, process.env.CACHED_BUILD)
```

Comment:
This might give you some trouble if your testing locally as everywhere else we are not relying on an environmental variable but on "cached" being in the arguments.

Reply:
I wrote a `.env` file to make my life a bit easier. 
I triggered the build multiple times and all boards appear without any problem. If you find no boards. force refresh the page on chrome it's` ctrl + F5 `